### ISSUE TITLE
[BUG] : Custom platforms/environments reverting to default after one playthrough. 

### ISSUE THREAD

**Issue by elysiankisaiya** [Zagrios/bs-manager#403]
## Bug Description
<!Custom platforms/environments reverting to default after one playthrough.>

## System Specs
[2024.01.19.12.13.33.log](https://github.com/Zagrios/bs-manager/files/13989039/2024.01.19.12.13.33.log)

**Comment by elysiankisaiya** [comment]
Not sure why the descriptions aren't showing on the report but this is the problem: 
The custom maps I've downloaded aren't showing the custom environments that match.. I played "aespa - Dreams Come True" mapped by eurnkung and for the first playthrough the environment was there and then on fail it switched back to default and has stayed on default. Clicking 'okay' in the in-game settings seems to reset it but it will still only allow one playthrough before it reverts back to default. I'm playing on v1.29.1 through bsmanager.

**Comment by elysiankisaiya** [comment]
Not sure why the descriptions aren't showing on the report but this is the problem:
The custom maps I've downloaded aren't showing the custom environments that match.. I played "aespa - Dreams Come True" mapped by eurnkung and for the first playthrough the environment was there and then on fail it switched back to default and has stayed on default. Clicking 'okay' in the in-game settings seems to reset it but it will still only allow one playthrough before it reverts back to default. I'm playing on v1.29.1 through bsmanager.
